docs/Basics-Component-ListView.md

```diff
@@ -20,20 +20,21 @@ This example creates a simple `ListView` of hardcoded data. It first initializes
 > A `rowHasChanged` function is required to use `ListView`. Here we just say a row has changed if the row we are on is not the same as the previous row.
 
 ```JavaScript
-import React from 'react';
+import React, {Component} from 'react';
```

Comment:
just to be consistent can you please add spaces around Component? 
`import React, { Component } from 'react';`